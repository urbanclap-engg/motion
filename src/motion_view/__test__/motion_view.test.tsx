import * as React from 'react';
import { render } from '@testing-library/react-native';
import { View } from 'react-native';
import MotionView from '../motion_view';

beforeEach(() => {
	jest.resetAllMocks();
});

describe('motion_view', () => {
	it('should render', () => {
		const { getByTestId, toJSON } = render(
			<MotionView>
				<View style={{ width: 100, height: 100, backgroundColor: 'red' }} />
			</MotionView>,
		);
		expect(getByTestId('motion_view')).toBeTruthy();
		const tree = toJSON();
		expect(tree).toMatchSnapshot();
	});
});
