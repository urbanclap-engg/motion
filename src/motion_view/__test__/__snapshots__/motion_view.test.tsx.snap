// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`motion_view should render 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      undefined,
      Object {
        "opacity": 0,
      },
      Object {},
      Object {},
      Object {},
    ]
  }
  testID="motion_view"
>
  <View
    style={
      Object {
        "backgroundColor": "red",
        "height": 100,
        "width": 100,
      }
    }
  />
</View>
`;
